test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenCreateSpecs.java

```diff
@@ -367,6 +368,22 @@ public HapiApiSpec creationRequiresAppropriateSigs() {
 				);
 	}
 
+	public HapiApiSpec creationRequiresAppropriateSigsHappyPath() {
+		return defaultHapiSpec("CreationRequiresAppropriateSigsHappyPath")
+				.given(
+						cryptoCreate("payer"),
+						cryptoCreate(TOKEN_TREASURY).balance(0L),
+						newKeyNamed("adminKey")
+				).when().then(
+						tokenCreate("shouldntWorkEither")
```

Comment:
looks good other than the name of this token should be "shouldWork" or anything appropriate

Reply:
Agree. Changed.